examples/experimental/snowflake_feedbacks.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        def generate_completion(self, query: str, context_str: list) -&gt; str:
</pre><p>can you update this to call Complete REST API?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>